### ISSUE TITLE
[Bug]: Every frame is completely different than the last

### ISSUE THREAD

**Issue by Trimad** [continue-revolution/sd-webui-animatediff#38]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### Have you read FAQ on README?

- [X] I have updated WebUI and this extension to the latest version

### What happened?

Every frame is completely different than the last
![00004-3142338252](https://github.com/continue-revolution/sd-webui-animatediff/assets/13515006/4bc9c8ef-5f6f-4ebd-aa4b-b29ae8a81a77)

### Steps to reproduce the problem

1. Go to .... 
2. Press ....
3. ...

### What should have happened?

It seems like every frame is a new seed, and then they're stitched together into a .gif.

### Commit where the problem happens

webui: 
extension: 

### Command Line Arguments

```Shell
@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS= --autolaunch --medvram --no-half-vae --no-half
call webui.bat
```

### Console logs

```Shell
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Creating model from config: C:\Users\Tristan\Documents\GitHub\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Startup time: 9.6s (launcher: 2.2s, import torch: 2.8s, import gradio: 0.7s, setup paths: 0.7s, other imports: 0.7s, list SD models: 0.2s, load scripts: 1.3s, create ui: 0.6s, gradio launch: 0.4s).
Applying attention optimization: Doggettx... done.
Model loaded in 2.9s (load weights from disk: 0.8s, create model: 0.4s, apply weights to model: 1.2s, calculate empty prompt: 0.4s).
2023-07-27 12:45:08,482 - AnimateDiff - INFO - AnimateDiff process start with video Max frames 16, FPS 8, duration 2.0,  motion module mm_sd_v15.ckpt.
2023-07-27 12:45:08,482 - AnimateDiff - INFO - Loading motion module mm_sd_v15.ckpt from C:\Users\Tristan\Documents\GitHub\stable-diffusion-webui\extensions\sd-webui-animatediff\model\mm_sd_v15.ckpt
2023-07-27 12:45:11,593 - AnimateDiff - WARNING - Missing keys <All keys matched successfully>
2023-07-27 12:45:11,899 - AnimateDiff - INFO - Hacking GroupNorm32 forward function.
2023-07-27 12:45:11,899 - AnimateDiff - INFO - Injecting motion module mm_sd_v15.ckpt into SD1.5 UNet input blocks.
2023-07-27 12:45:11,900 - AnimateDiff - INFO - Injecting motion module mm_sd_v15.ckpt into SD1.5 UNet output blocks.
2023-07-27 12:45:11,900 - AnimateDiff - INFO - Injection finished.
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:38<00:00,  1.95s/it]
2023-07-27 12:45:56,907 - AnimateDiff - INFO - Removing motion module from SD1.5 UNet input blocks.35<00:00,  1.86s/it]
2023-07-27 12:45:56,907 - AnimateDiff - INFO - Removing motion module from SD1.5 UNet output blocks.
2023-07-27 12:45:56,907 - AnimateDiff - INFO - Restoring GroupNorm32 forward function.
2023-07-27 12:45:56,908 - AnimateDiff - INFO - Removal finished.
2023-07-27 12:45:56,908 - AnimateDiff - INFO - Merging images into GIF.
2023-07-27 12:45:57,986 - AnimateDiff - INFO - AnimateDiff process end.
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:42<00:00,  2.13s/it]
2023-07-27 12:46:12,265 - AnimateDiff - INFO - AnimateDiff process start with video Max frames 16, FPS 8, duration 2.0,  motion module mm_sd_v15.ckpt.
2023-07-27 12:46:12,926 - AnimateDiff - INFO - Hacking GroupNorm32 forward function.
2023-07-27 12:46:12,926 - AnimateDiff - INFO - Injecting motion module mm_sd_v15.ckpt into SD1.5 UNet input blocks.
2023-07-27 12:46:12,927 - AnimateDiff - INFO - Injecting motion module mm_sd_v15.ckpt into SD1.5 UNet output blocks.
2023-07-27 12:46:12,928 - AnimateDiff - INFO - Injection finished.
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:38<00:00,  1.92s/it]
2023-07-27 12:46:57,213 - AnimateDiff - INFO - Removing motion module from SD1.5 UNet input blocks.35<00:00,  1.86s/it]
2023-07-27 12:46:57,213 - AnimateDiff - INFO - Removing motion module from SD1.5 UNet output blocks.
2023-07-27 12:46:57,214 - AnimateDiff - INFO - Restoring GroupNorm32 forward function.
2023-07-27 12:46:57,214 - AnimateDiff - INFO - Removal finished.
2023-07-27 12:46:57,215 - AnimateDiff - INFO - Merging images into GIF.
2023-07-27 12:46:58,530 - AnimateDiff - INFO - AnimateDiff process end.
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:42<00:00,  2.12s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:42<00:00,  1.86s/it]
```

**Comment by Trimad** [comment]
I figured it out. Don't but BREAK in your prompt because the number of tokens can't be above 75.

**Comment by StableInfo** [comment]
I am getting this (2 types of different images, 8 frames each indeed)
But this does not ALWAYS happen:
![00046-1852841998](https://github.com/continue-revolution/sd-webui-animatediff/assets/136593585/bba50f7d-09c2-4254-a048-2ac2ef0061cf)

and a better one:
![00047-1852841998](https://github.com/continue-revolution/sd-webui-animatediff/assets/136593585/a9ab25fc-c341-448c-a2ba-26601854f86c)

**Comment by StableInfo** [comment]
![image](https://github.com/continue-revolution/sd-webui-animatediff/assets/136593585/c1ff9aea-f258-4ec4-b9b0-c75f5ea3e4e6)

**Comment by Casuallynoted** [comment]
I'm also getting this bug. I've noticed that the first several times I did it it worked perfectly, and it output a preview like the one in the post above. Now when I do it, it only shows one image in the preview and every image in the resulting animation is totally different. I don't believe I've changed anything in my settings enough to cause this.

**Comment by Ainaemaet** [comment]
Also getting every frame is a completely separate image. 

There is no cohesion whatsoever. 

Extension worked fine previously, no idea what's causing it.

**Comment by Ainaemaet** [comment]
Ok, for me the problem turned out to be missing the v14 motion model from the extension models folder. It looks like somehow it got changed somewhere and it only downloaded v15 correctly. :)

**Comment by StableInfo** [comment]
Where is it?

**Comment by Casuallynoted** [comment]
I have mine in the correct place (and don't get an error about anything missing, to make sure) so I don't think that's what's causing it in my case.

**Comment by Ainaemaet** [comment]
> 

I never got an error either surprisingly. It just didn't work and I happened to think "I wonder if it's an issue with the motion module not being downloaded properly or something". 

Somewhere along the way the program had changed where it looks for them and in the new location it had only downloaded v15 (which I hadn't even tried to use). 

@StableInfo mine are located in /sd-webui/extensions/sd-webui-animatediff/model/

**Comment by Casuallynoted** [comment]
Just tried reproducing the issue on several machines and found a commonality between them:

On a fresh install, they seem to work great UNTIL the CUDA memory is exceeded. Once that error has popped up, any future AnimateDiff generations will be total nonsense. The option to clear the memory module doesn't seem to help with this. The issue persists after device reboots.

**Comment by StableInfo** [comment]
Why "completed"? The issue

**Comment by continue-revolution** [comment]
Does such problem still exist if you update everything and enable "Pad cond uncond" in Settings/Optimizations? I never meet this issue and I'm not sure if it still exists.

**Comment by StableInfo** [comment]
Sorry I can't test it right now. Will make sure to rewrite a comment if it happens to me again
